modules/api/src/main/java/org/eclipse/xpanse/api/OrchestratorApi.java

```diff
@@ -417,4 +422,9 @@ public String openApi(@PathVariable("id") String id) {
         return apiUrl;
     }
 
+    private String generateCustomerServiceName(CreateRequest createRequest) {
+        return createRequest.getServiceName().substring(0, 4) + "-"
```

Comment:
what will happen if the service name is shorter than 4?

Reply:
@iskey good point. Fixed this.